ui/app/components/job-status/panel/steady.js

```diff
@@ -246,7 +246,7 @@ export default class JobStatusPanelSteadyComponent extends Component {
     }
 
     const healthyAllocs = this.allocBlocks.running?.healthy?.nonCanary;
-    if (healthyAllocs?.length === totalAllocs) {
+    if (healthyAllocs?.length && healthyAllocs?.length === totalAllocs) {
```

Comment:
Specifying that "0 allocs running of your 0 expected allocations!" doesn't mean "healthy"